src/routes/categories/containers/CategoriesContainer.js

```diff
@@ -49,5 +52,12 @@ const mapStateToProps = (state: StateType, ownProps) => {
 
 // $FlowFixMe
 const themed = withTheme(props => <ScrollView><Categories {...props} /></ScrollView>)
-// $FlowFixMe connect()
-export default withRouteCleaner(connect(mapStateToProps, mapDispatchToProps)(themed))
+export default compose( // $FlowFixMe connect()
```

Comment:
Is that $FlowFixMe still supressing the same issue as before? I'm a bit concerned because of the branch thing because Flow fails a lot with it. If that is the case just add an other $FlowFixMe with a link to a issue maybe.

Reply:
I could remove all flowfixmes.